### ISSUE TITLE
The default lookup path of .env is not documented

### ISSUE THREAD

**Issue by hakilebara** [fivetanley/ember-cli-dotenv#65]
By default `.env` is expected to by located in the `config`folder instead of the root folder of the app. Is this intentional? 

```js
// config/dotenv.js
/* eslint-env node */

'use strict';

const path = require('path');

module.exports = function(/* env */) {
  return {
    clientAllowedKeys: [],
    fastbootAllowedKeys: [],
    failOnMissingKey: false,
    path: path.join(__dirname, '.env')
  }
};
```

**Comment by SergeAstapov** [comment]
@hakilebara it is definitely not, thank you for reporting! Created #66 to address this

**Comment by SergeAstapov** [comment]
Fix released in v2.2.3 and v3.0.0
